### DISCUSSION TITLE
Macbook para esta carrera

### DISCUSSION THREAD

**Discussion by paulavmunoz** [hackatomik/SISCOx256#39] [Chill | Carrera]
En el caso tal de ya contar una mac, qué significaría para el transcurso de la carrera? Es decir, cuánto me serviría y cuánto no. Es la mejor opción conseguir una PC directamente o hay some wiggle room?

**Comment by quantumquirkxyz** [comment]
Tener una MacBook para la carrera de Ingeniería de Sistemas y Computación es una excelente opción, ya que ofrece un entorno basado en Unix, ideal para programación y desarrollo de software, con herramientas como Xcode y lenguajes como Python, C, y Java. Sin embargo, algunos programas especializados en ingeniería pueden estar optimizados para Windows, por lo que en esos casos podrías usar Boot Camp o virtualización para instalar Windows. Si bien las MacBooks son más caras, su rendimiento y estabilidad compensan el costo si ya tienes una. Además, es altamente recomendable usar una máquina virtual para aprender Linux, ya que te permite experimentar con un sistema operativo utilizado en servidores y entornos de desarrollo sin afectar tu máquina principal, dándote flexibilidad y experiencia valiosa para la carrera.

**Reply by tapilew** [reply]
Te ahorras el trabajo de WSL haha
